docs/TUTORIAL.md

```diff
@@ -1,3 +1,163 @@
 # Tutorial
 
-Coming soon!
+# Tutorial
+
+The **ksonnet** [readme][readme] 
```

Comment:
Might be worth considering making the value prop a little clearer up front to contextualize why this is important. Straw proposal to brainstorm:

```
ksonnet allows users to trivially add sidecars to Kubernetes applications (_e.g._, adding fluentd, linkerd, or istio) using _mixin libraries_. In this tutorial, we will:

* Introduce the concept of a mixin.
* Demonstrate how to write a mixin library that adds fluentd to a deployment object.
* Demonstrate how to use the mixin library in Kubernetes application code.
```

If we agree about the general structure of that intro, it would also seem to imply that we should structure the document differently, to concentrate less on building the deployment object up from scratch.

Thoughts?